cypress/support/pages/signIn.pageObject.js

```diff
@@ -25,6 +25,15 @@ class SignInPageObject extends PageObject {
       .type(password);
   }
 
+  fillLoginCredentials(email, password) {
+    if (email) {
+      this.typeEmail(email);
```

Comment:
You are calling 'typeEmail' and 'typePassword' methods without ensuring that they exist and are functions. It's a good practice to check if these are valid functions before calling them to avoid potential errors at runtime.

Reply:
Done